recipes/opensubdiv/all/conanfile.py

```diff
@@ -0,0 +1,165 @@
+import functools
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
```

Comment:
```suggestion
from conan import ConanFile
from conan.tools.files import get, replace_in_file
from conans import CMake, tools
from conan.errors import ConanInvalidConfiguration

required_conan_version = ">=1.48.0"

```
Prepare for Conan 2.x